docs/node-field-selector.md

```diff
@@ -0,0 +1,42 @@
+# Node Field Selectors
+
+![GA](assets/ga.svg)
```

Comment:
I think this should be `alpha` or `beta`.

Reply:
I'd say `alpha`